Assets/Scripts/Bullets/BulletController.cs

```diff
@@ -1,24 +1,39 @@
-﻿using UnityEngine;
+﻿using System.Collections;
+using UnityEngine;
 
 public class BulletController : MonoBehaviour
 {
+    [SerializeField]
+    private ParticleSystem explosionParticle;
+    [SerializeField]
+    private AudioSource bulletExplosionAudio;
+
+
+
     [SerializeField]
     private float fireSpeed = 20f;
 
 
+
     public void Fire(Transform fireTransform, GameObject tank)
     {
-        
+
         GameObject bullet = Instantiate(gameObject, fireTransform.position, fireTransform.rotation);
```

Comment:
Bullet Service and Instantiate the bullets there and get them when required (Same way you do for tanks)